jest.config.js

```diff
@@ -59,7 +59,12 @@ const projectForPlatform = platform => {
     // Config for finding and transforming source code.
     //
 
-    testPathIgnorePatterns: ['/node_modules/', '/src/__tests__/lib/', '-testlib.js$'],
+    testPathIgnorePatterns: [
+      '/node_modules/',
+      '/src/__tests__/lib/',
+      '-testlib.js$',
+      '/types/__tests__',
```

Comment:
How about putting the tests under `types/__flow-tests__/` instead? Like our existing `src/__flow-tests__/`.

Then the name won't look in the first place like it's something Jest should be running, so this line shouldn't be needed. Also the eslintignore line won't be needed, because there's already one for `**/__flow-tests__/**`.

Reply:
> Also the eslintignore line won't be needed, because there's already one for `**/__flow-tests__/**`.

Hmm I'm not sure which line this is referring to.